src/locales/zh-CN/component.ts

```diff
@@ -33,4 +34,9 @@ export default {
   'component.ssl.fieldKeyInvalid': '请检查 Key 值',
   'component.ssl.fieldCertInvalid': '请检查 Cert 值',
   'component.ssl.invalidKey': '非法的 Key',
+  'component.ssl.fieldKeyTooShort': 'Key 值过短，至少需要128位',
+  'component.ssl.fieldCertTooShort': 'Cert 值过短，至少需要128位',
+  'component.ssl.sni': 'SNI',
```

Comment:
Because SNI, Key is always shown in English, so there has no need to do i18n for them :)

Reply:
Ok, I will change them.